components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.cpp

```diff
@@ -136,10 +137,11 @@ class RFPins {
 RFPins::RFPins(PinName spi_sdi, PinName spi_sdo,
                PinName spi_sclk, PinName spi_cs, PinName spi_sdn,
                PinName spi_gpio0, PinName spi_gpio1, PinName spi_gpio2,
-               PinName spi_gpio3)
+               PinName spi_gpio3, PinName ant_sel)
```

Comment:
Could this change be made so that ant_sel is given as parameter in configure_exernal_frontend

Could this then be backward compatible change?

Reply:
Sure I'll make the change